.golangci.yaml

```diff
@@ -242,6 +242,9 @@ linters:
       - linters:
           - lll
         source: "^// http "
+      - linters:
+          - staticcheck
+        text: "SA1019.*shared.ManagedBy is deprecated"
```

Comment:
For the reviewer as we introduced the deprecation of the managedBy label. The linter claims of still usage of the managedBy which are not related to the watcher CR. For now , supressing the deprecation message only temporary and will be removed on the follow up ticket.